Makefile

```diff
@@ -31,3 +31,21 @@ tag:
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	docker build -t ${IMG} .
+
+sbom-tool:
+	curl -Lo sbom-tool https://github.com/microsoft/sbom-tool/releases/latest/download/sbom-tool-linux-x64
```

Comment:
A pity that they don't sign linux binaries yet...

Reply:
Yes, but I think it's on the way: https://github.com/microsoft/sbom-tool/issues/44